bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Field.java

```diff
@@ -98,4 +98,14 @@ default Object[] getDefault() {
         return null;
     }
 
+    /**
+     * The expression that when evaluated would determine what the displayValue of a field would be
+     *
+     * @return display value expression of the field
+     * @since com.adobe.cq.forms.core.components.models.form 5.2.0
+     */
+    default String getDisplayValueExpression() {
```

Comment:
Please add a test case for the default method

Reply:
Done.